modules/wkt/src/lib/encode-wkb.ts

```diff
@@ -33,7 +33,7 @@ enum WKB {
 /**
  * Options for encodeWKB
  */
-interface WKBOptions {
+type WKBOptions = {
```

Comment:
Is `type` better than `interface`?

Reply:
If not better, it is at least less ambiguous. Admittedly I have some baggage from other languages, but to me, `interface` is an object oriented concept that is intended to describe (mostly) methods that need to be defined by a class, so I think it looks strange to see `interface` used to describe a pure object without any methods. 

I suspect `interface` is favored in some type script code because of historical reasons (maybe `type` was a later addition?)